elastic/integrations
# [Elastic Agent] Fix agent dashboard links for non default spaces

## What does this PR do?

Referencing dashboards with markdown needs to be space-aware. Currently it was using a hyperlink that would always choose the default space on many occasions, the PR simply changes the markdown links to the correct format.

Tested all the links on all the dashboards after the changes, and they are working fine.

## Checklist


## How to test this PR locally

Elastic Agent integration is installed by default, open up the Dashboards UI and click on the Agent Overview dashboard, then click on any of the links in the table of contents on the left, this should now work in any space for all the links.

## Related issues

- Closes #6691
